### ISSUE TITLE
[bug] Expected a valid CSS identifier

### ISSUE THREAD

**Issue by djkato** [simeydotme/svelte-range-slider-pips#123]
**Describe the bug**
node_modules/svelte-range-slider-pips/src/RangeSlider.svelte:686:73 Expected a valid CSS identifier

**To Reproduce**

create new astro project, install svelte (Installs 5.0.0-next.1), try to use like:
```jsx
<RangeSlider {max} {min} step="1" pushy={true} values={[50]} />
```
**Screenshots**
![image](https://github.com/simeydotme/svelte-range-slider-pips/assets/25299243/679baade-877d-403e-bb45-7da583de5e41)

**Comment by serferdinand2** [comment]
> **Describe the bug** node_modules/svelte-range-slider-pips/src/RangeSlider.svelte:686:73 Expected a valid CSS identifier
> 
> **To Reproduce**
> 
> create new astro project, install svelte (Installs 5.0.0-next.1), try to use like:
> 
> ```js
> <RangeSlider {max} {min} step="1" pushy={true} values={[50]} />
> ```
> 
> **Screenshots** ![image](https://private-user-images.githubusercontent.com/25299243/287099391-679baade-877d-403e-bb45-7da583de5e41.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYxMzAzMzcsIm5iZiI6MTcwNjEzMDAzNywicGF0aCI6Ii8yNTI5OTI0My8yODcwOTkzOTEtNjc5YmFhZGUtODc3ZC00MDNlLWJiNDUtN2RhNTgzZGU1ZTQxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTI0VDIxMDAzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI1NTIzMzZjMzFjOTNhOWI4MDUyYmY3NjVmOWRmMzdlNTYyMzljZmZlYTJhNDYxNjFiNzliOTg2ZTdiMTkyY2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.kO_YorpsSiozgooKzsp_cRxaB90-Wpon0SkPpt3wwJk)



I believe this was fixed in the latest release https://github.com/sveltejs/svelte/releases/tag/svelte%405.0.0-next.38

**Comment by simeydotme** [comment]
oh it was actually a bug in Svelte, not a design change? :D 
thanks for sharing that!

**Comment by serferdinand2** [comment]
I might be wrong about this but |global has issues in svelte 5. Right now it deletes the whole DOM on global transitions..

**Comment by simeydotme** [comment]
Should be resolved by #145
